src/Graphula/Node.hs

```diff
@@ -134,6 +135,7 @@ nodeKeyed
      , PersistEntityBackend a ~ SqlBackend
      , PersistEntity a
      , Typeable a
+     , GraphulaSafeToInsert a
```

Comment:
so, technically, this is a change to the signature of the function, which is a breaking change

but also technically, it's only a breaking change if `persistent` itself is updated to `2.14`. Otherwise this evaluates out to a `()` and should discharge

I can also do another approach where I CPP this in at each use-site, which will prevent the API from changing for users unless they are on `persistent-2.14` anyway.

Reply:
I'm personally OK with us only doing a minor bump ourselves given this nuance. Curious what other Frecklers thing.

I don't think peppering the CPP around is worthwhile. I think we can either bend the rules or just do a major bump.